src/bindbc/hb/dynload.d

```diff
@@ -286,7 +286,7 @@ HBSupport loadHarfBuzz(const(char)* libName)
 	lib.bindSymbol(cast(void**)&hb_font_set_var_coords_design, "hb_font_set_var_coords_design");
 	lib.bindSymbol(cast(void**)&hb_font_set_var_coords_normalized, "hb_font_set_var_coords_normalized");
 	lib.bindSymbol(cast(void**)&hb_font_get_var_coords_normalized, "hb_font_get_var_coords_normalized");
-	lib.bindSymbol(cast(void**)&hb_font_set_var_named_instance, "hb_font_set_var_named_instance");
+	//lib.bindSymbol(cast(void**)&hb_font_set_var_named_instance, "hb_font_set_var_named_instance");
```

Comment:
I am not sure this is in the latest harfbuzz, or at least it was not in the version of harffbuzz that i get from vspkg . . . Didnt really mean for it to get included in commit

Reply:
Any symbol that is deprecated or introduced in a newer version than a reasonable one in e.g. Debian stable should allow for the error count to increase without forcing the load to fail.